src/QSymbolicsBase/basic_ops_inhomogeneous.jl

```diff
@@ -42,31 +73,38 @@ operation(x::SBraKet) = *
 head(x::SBraKet) = :*
 children(x::SBraKet) = [:*,x.bra,x.ket]
 Base.:(*)(b::Symbolic{AbstractBra}, k::Symbolic{AbstractKet}) = SBraKet(b,k)
-function Base.show(io::IO, x::SBraKet)
-    print(io,x.bra)
-    print(io,x.ket)
-end
+Base.show(io::IO, x::SBraKet) = begin print(io,x.bra); print(io,x.ket) end
 
 """Symbolic application of a superoperator on an operator"""
-@withmetadata struct SApplyOp <: Symbolic{AbstractOperator}
```

Comment:
This being renamed worries me. Any specific reason not to stick to the previous name? It is not a public name, so it is not too breaking, but I want to have a good reason to do it.

Reply:
My worry is that `SApplyOp` sounds like it is symbolic multiplication of operators, rather than application of a superoperator on an operator.

Reply:
You are right about that. How about `SSuperOpApply`? My goal is to distinguish it from the others who follow the convention `SApply<the thing being acted upon>` and not `SApply<the thing that is acting>`.